package.json

```diff
@@ -58,7 +58,7 @@
     "@angular/platform-server": "~7.0.3",
     "@angular/router": "~7.0.3",
     "@angular/upgrade": "~7.0.3",
-    "@hmcts/ccd-case-ui-toolkit": "2.64.7-EUI-CCD-RDM-8661-MoneyGBP",
+    "@hmcts/ccd-case-ui-toolkit": "2.64.7-EUI-CCD-CCD-1047-Underscore-Import",
```

Comment:
@skandala-cgi  - It would be better to get a stable release from ui toolkit after the merge of its PR, and use that stable release here. I've just merged that PR. https://github.com/hmcts/ccd-case-ui-toolkit/pull/807